### ISSUE TITLE
Support two sets of intervals: security updates live, other updates {daily,weekly,monthly}

### ISSUE THREAD

**Issue by killercup** [dependabot/feedback#102]
Would it be possible to a separate configuration for security updates?

I've just added dependabot to a Rust project of mine and talked with the maintainers about the settings to use. We have two requirements: We'd like to get security updates quickly, and we don't want to get a huge bunch of PRs for non-critical updates.

If dependabot had a setting "Create PRs for security critical updates immediately", and otherwise use the "Monthly updates" setting, we'd be quite happy.

**Comment by greysteil** [comment]
Thanks for getting in touch and for giving Dependabot a try.

If you set your update setting to monthly that should do the trick. For security updates we *always* create the PRs immediately (because we can't imagine a use case where you wouldn't want that), so Dependabot should do exactly what you want with that setup.

**Comment by greysteil** [comment]
Agreed - we could make the frontend way clearer on some of these things.
